composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
Let's not rule out old PHP versions if they work with phpbrake. I suggest to close this PR and update min version requirements only when we know for sure that PHP version X.X.X is not supported by phpbrake.